app/static/event/check_country_time.js

```diff
@@ -0,0 +1,79 @@
+const ERROR_TIME_DURATION = 3000;
+const TIME_NOT_FILLED = 'Please enter the meeting date and time';
+const COUNTRY_NOT_FILLED = 'Please choose country';
+const modal_container = document.querySelector('.countries-modal-container');
+const open_modal = document.querySelector('.open-countries-modal');
+const close_modal = document.querySelector('.close-countries-modal');
+const submit_country = document.querySelector('.check-time');
+const upper_result = document.querySelector('.upper-result');
+const start_result = document.querySelector('.start-result');
+const end_result = document.querySelector('.end-result');
+const error_msg = document.querySelector('.empty-fields-error');
+const user_timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
This should run only after the document has fully loaded.
Currently, if the page hasn't loaded yet, these lines might bring up errors.